metricbeat/module/system/filesystem/helper.go

```diff
@@ -112,8 +115,10 @@ func filterFileSystemList(fsList []sigar.FileSystem) []sigar.FileSystem {
 // GetFileSystemStat retreves stats for a single filesystem
 func GetFileSystemStat(fs sigar.FileSystem) (*FSStat, error) {
 	stat := sigar.FileSystemUsage{}
```

Comment:
I still wonder if we should log this somewhere. Maybe in the main `filesystem.go`?

Reply:
my worry is flooding the logs with this warning/error message, knowing folks will collect these metrics in short intervals. 
I would avoid it for now and keep it in the documentation. If we get any user feedback about any confusion there, then we might need to reconsider.